dll/win32/lpk/lpk.c

```diff
@@ -81,16 +78,30 @@ LpkExtTextOut(
     /* Check if the string requires complex script processing and not a "glyph indices" array */
     if (ScriptIsComplex(lpString, uCount, SIC_COMPLEX) == S_OK && !(fuOptions & ETO_GLYPH_INDEX))
     {
+        LPWORD glyphs = NULL;
+        LPWSTR reordered_str = HeapAlloc(GetProcessHeap(), 0, uCount * sizeof(WCHAR));
+        INT cGlyphs;
+        BOOL bResult;
+
         BIDI_Reorder(hdc, lpString, uCount, GCP_REORDER,
```

Comment:
Why ignore the return from this?

Reply:
Wine also does ignore the return value of the function, should I add an escape codepath in case of failure?

Reply:
You should ask yourself, what could happen if this function call fails: what should the code following the call do, etc...
Maybe it's interesting to retrieve the return value of the function to at least print some warning in case of failure?